### PR TITLE
GrapheneOS apps

### PR DESCRIPTION
**Pull request by shuvashish76**
https://play.google.com/store/apps/dev?id=7626572346998944846
All these 3 apps' apk available on GitHub with different package ID

Edit: Reminder : Don't forget to add new "Emulator"  to Table of contents for both Apps & Games section. Added with #216 PR

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,8 +154,10 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 ### • Camera
 
 * [**Open Camera**](http://opencamera.sourceforge.net/) <sup>**[[F-Droid](https://f-droid.org/app/net.sourceforge.opencamera)]**</sup>
+  * [**OpenCamera Sensors**](https://github.com/MobileRoboticsSkoltech/OpenCamera-Sensors)
```
Comment (offa) [inline README.md:157]: This looks based on *OpenCamera*, it even contains it's F-Droid link :rofl:

**Review by offa — APPROVED** [review]

**Comment by offa** [comment]
Thanks! I wonder why none of the GrapheneOS projects is on F-Droid.

There's an indention issue reported by CI, otherwise thumbs-up! :+1:  :-)



> Edit: Reminder : Don't forget to add new "Emulator" to Table of contents for both Apps & Games section. Added with #216 PR

Thanks for pointing this out, seems I have missed the ToC :man_facepalming: .

File: README.md (hunk 1 above)
Comment (shuvashish76) [reply README.md]: > This looks based on OpenCamera

Yes, with some extra features...

> even contains it's F-Droid link

F-Droid link ??

**Comment by shuvashish76** [comment]
> I wonder why none of the GrapheneOS projects is on F-Droid.

Please check all these comments here: https://archive.fo/OtTGX
All comments of Daniel Micay (_Founder and lead developer of GrapheneOS_) were deleted by F-Droid, see here: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/9620#note_856255961

> There's an indention issue reported by CI, otherwise thumbs-up!

Fixed thanks.

**Review by offa — APPROVED** [review]

**Comment by offa** [comment]
> > I wonder why none of the GrapheneOS projects is on F-Droid.
> 
> Please check all these comments here: https://archive.fo/OtTGX All comments of Daniel Micay (_Founder and lead developer of GrapheneOS_) were deleted by F-Droid, see here: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/9620#note_856255961
> 
> > There's an indention issue reported by CI, otherwise thumbs-up!
> 
> Fixed thanks.

Oh boy ... that explains why ... 🤦‍♂️

File: README.md (hunk 1 above)
Comment (offa) [reply README.md]: Its left in a Html document: https://github.com/MobileRoboticsSkoltech/OpenCamera-Sensors/blob/d7df21d0ab533162e5ca7d4fa7c1a3908065402c/_docs/info.html#L106

😄

**Comment by offa** [comment]
Thank you! :-)

File: README.md (hunk 1 above)
Comment (offa) [reply README.md]: Thanks, Update is on the way …

**Pull request merged**